Fix build error produced by SwiftLint for 'force_try'

Takuro-Ito opened the pull request:
## Notes
- **This pull request resolves a build error produced by `SwiftLint`.**
- ***Warning*** is now produced instead of ***Error*** by `SwiftLint`.

### Before
<img width="1269" alt="image" src="https://user-images.githubusercontent.com/12025248/114264514-5fb40480-9a26-11eb-95cb-f081c7bdb0cb.png">

### After
<img width="1269" alt="image" src="https://user-images.githubusercontent.com/12025248/114264913-b4587f00-9a28-11eb-8351-1fad3dc5f7d5.png">

Takuro-Ito committed:
Fix build error produced by SwiftLint for 'force_try'

reitermarkus commented:
Can only this line be ignored? This shouldn't be a warning since the regex is built from a static string which should always succeed.

Takuro-Ito committed:
Revert .swiftlint.yml and Disable 'force_try' for a single line

Takuro-Ito commented:
Yes and I updated my pull request.
I just knew SwiftLint supports for this kind of situation. Details: https://github.com/realm/SwiftLint#disable-rules-in-code

## Changes
- Reverted `.swiftlint.yml` to the original one
- Disabled a `force_try` rule for a single line

reitermarkus commented:
Thank you, @Takuro-Ito!
